plugins/ata/fu-self-test.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupd.h>
+
+#include "fu-ata-device.h"
+#include "fu-test.h"
+
+static void
+fu_ata_id_func (void)
+{
+	gboolean ret;
+	gsize sz;
+	g_autofree gchar *data = NULL;
+	g_autofree gchar *path = NULL;
+	g_autoptr(FuAtaDevice) dev = NULL;
+	g_autoptr(GError) error = NULL;
+
+	path = fu_test_get_filename (TESTDATADIR, "StarDrive-SBFM61.2.bin");
```

Comment:
Rather than include a fully non-free binary, can you just DD out the relevant bits that are checked and set it to 0x0 on the rest?

Reply:
It's not firmware, it's the output of the IDENTIFY glob. It's like the EDID for a monitor.

Reply:
Ah OK.